### PR TITLE
Component branch

### PR DESCRIPTION
**Pull request by tjtemple**
Extremely basic component with simple css that hooks up to the API call to display all trains. Lays the groundwork to add more information and style to the basic component
image

### CODE REVIEW

File: train-tracker/src/TrainList.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    // use once pattern
+    React.useEffect(() => {
+        trainData.onUpdated = function() {
+            setTrainList(this.trains);
+        }
+        trainData.update();
+    },[])
+    
+    //console.log(trainList);
```
Comment (molly-pop) [inline train-tracker/src/TrainList.js:47]: I would just scrub out this comment in te future! This is a nitpick though

**Review by molly-pop — COMMENTED** [review]
Looks good! I left one nitpick over a comment, but honestly looks great. Liked the test trains

File: train-tracker/src/TrainList.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                </div>
+            </div>
+            
+        )
+    } // end MakeTrain
+
+    const [trainList, setTrainList] = React.useState([]);
+    const trainData = new Amtrak.APIInstance();
+
+    // use once pattern
```
Comment (ldups) [inline train-tracker/src/TrainList.js:39]: think we could also get rid of this comment, and the // end MakeTrain one

File: train-tracker/src/index.js [hunk 3]
```diff
@@ -3,15 +3,16 @@ import ReactDOM from 'react-dom/client';
...
+import Amtrak, { Train } from './AmtrakAPI'
 import Map from './Map';
+import TrainList from './TrainList';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App/>
     <Map/>
-
```
Comment (ldups) [inline train-tracker/src/index.js:14]: In the future we'll want the Map component and the TrainList component within the App component, but obviously an easy fix and not a big deal rn

**Review by wSimsT — APPROVED** [review]
Looks good

**Review by ahgoldmeer — COMMENTED** [review]
As others said, we should talk soon about how we want to handle API calls, search returns, etc. That said, this looks great!

**Review by ahgoldmeer — APPROVED** [review]

**Pull request merged**